rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/selection/ScriptExecutor.java

```diff
@@ -77,6 +77,7 @@ private Node executeScripts() {
         boolean selectionScriptSpecified = selectionScriptList != null && selectionScriptList.size() > 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1155)